arch/x86/x86_functions.h

```diff
@@ -89,6 +92,8 @@ uint32_t crc32_vpclmulqdq(uint32_t crc32, const uint8_t *buf, size_t len);
 #      define native_longest_match longest_match_sse2
 #      undef native_longest_match_slow
 #      define native_longest_match_slow longest_match_slow_sse2
+#      undef native_crc32
+#      define native_crc32 crc32_chorba_sse
```

Comment:
```c
#ifdef X86_CHORBA_SSE_CRC
```
Because the function is defined only if macro ``X86_CHORBA_SSE_CRC`` is defined.